core/dbt/exceptions.py

```diff
@@ -236,7 +236,8 @@ class VersionsNotCompatibleException(SemverException):
 
 
 class NotImplementedException(Exception):
-    pass
+    CODE = 10010
```

Comment:
Hmm, not sure if we need / want to set these. I believe these CODE/MESSAGE attributes are only used in the `rpc` server, and I'm not exactly sure how we decide which types of exceptions get their own codes.

@beckjake what do you think?

Reply:
Yeah, I'd just leave them at the defaults as inherited from `Exception`. The decision process there is pretty ad-hoc: I ask myself "would it be valuable to know I got this error based on the error code if I were writing an RPC client?", and I think the answer here is "no", because I don't think we should ever get this error from the RPC server.